### DISCUSSION TITLE
Need help with my 6550 engraver - need first project to engrave

### DISCUSSION THREAD

**Discussion by Tbullter** [arkypita/LaserGRBL#1058] [Q&A]
I received the Yora Laser Engraver 6550 as a gift.
I know nothing about it.
I am  very excited to start using it, but know nothing about coding or any of this type of thing.
I have gotten it set up.
Now I am at "open file", but of course there is nothing in the file for me to open with "GCode" so I can start engraving.
Can someone help me out Please?
Is there something more I can download to get images in to my computer in GCode or Image File Format to get started?
Any help would be greatly appreciated!
Thankyou

**Comment by StuartB4** [comment]
You should be able to open any image file,  jpg, png, bmp, gif, svg vector files, nc
Just download any image from google images just to test it. 

When you click File, Open file and the box opens just click the little arrow to see what type you can load. 

![ppppppppp](https://user-images.githubusercontent.com/16980026/85239599-5a9caf00-b42c-11ea-8494-baa1d9b19bd0.jpg)
